src/main/scala/org/broadinstitute/dsde/workbench/avram/db/EntityComponent.scala

```diff
@@ -0,0 +1,50 @@
+package org.broadinstitute.dsde.workbench.avram.db
+
+
+import AvramPostgresProfile.api._
+//import org.broadinstitute.dsde.workbench.avram.api.Entity
+import spray.json._
+
+import scala.concurrent.ExecutionContext
+
+case class EntityRecord(id: Long, name: String, collection: Long, entityBody: JsValue)
+
+trait EntityComponent extends AvramComponent  {
+  this: CollectionComponent =>
+
+  class EntityTable(tag: Tag) extends Table[EntityRecord](tag, "entity") {
+    def id =             column[Long]      ("id", O.PrimaryKey, O.AutoInc)
+    def name =           column[String]    ("name", O.Length(254))
+    def collection =     column[Long]      ("collection")
+    def entityBody =     column[JsValue]   ("entityBody", O.Length(254), O.SqlType("JSONB"))
```

Comment:
Based on https://github.com/tminglei/slick-pg#configurable-typemappers, it looks like we could use circe instead of spray-json, which would change this `spray.json.JsValue` into `io.circe.Json`.